packages/panels/src/Panel/Concerns/HasTheme.php

```diff
@@ -63,4 +66,17 @@ public function getDefaultTheme(): Theme
     {
         return FilamentAsset::getTheme('app');
     }
+
+    public function defaultThemeMode(ThemeMode $mode): static
+    {
+        $this->defaultThemeMode = $mode;
+
+        return $this;
+    }
+
+    public function getDefaultThemeMode(): string
```

Comment:
Please do the ->value bit in the views, all methods should return the original enum